docs/releasing/publishing-a-pre-release.md

```diff
@@ -1,19 +1,31 @@
-# Publishing pre-release of GOV.UK Frontend
+# Publish pre-release of GOV.UK Frontend
 
-We use pre-releases when:
-- We are working on a new contribution and want to see it in the Design System website
-- We want to try an experimental feature as if it's pushed to npm.
+Before you use a pre-release, you need to make changes to GOV.UK Frontend. Then follow these instructions.
```

Comment:
```suggestion
This pre-release guidance is aimed at Design System team members. If you're an external contributor who needs to create a pre-release, please [contact the Design System team](https://design-system.service.gov.uk/get-in-touch/) and we'll do it for you.

Before you use a pre-release, you need to make changes to GOV.UK Frontend. Then follow these instructions.
```

Reply:
@hannalaakso ^ Suggested text if we're encouraging external contributors to ask us instead of creating the pre-release themselves.

Reply:
@EoinShaughnessy Yes I think this works 👍